benchmarks/000.microbenchmarks/030.clock-synchronization/python/function.py

```diff
@@ -3,19 +3,39 @@
 import socket
 from datetime import datetime
 from time import sleep
+from jsonschema import validate
 
 from . import storage
 
 def handler(event):
 
+    schema = {
+        "type": "object",
+        "required": [ "request_id", "server-address", "server-port", "repetitions", "output-bucket", "income-timestamp" ],
+        "properties": {
+            "request-id": {"type": "integer"},
+            "server-address": {"type": "integer"},
+            "server-port": {"type": "integer"},
+            "repetitions": {"type": "integer"},
+            "output-bucket": {"type": "object"},
+            "income-timestamp": {"type": "number"}
+        }
+    }
+    try:
+        validate(event, schema=schema)
+    except:
+        # !? To return 'measurement': {'bucket-key': None, 'timestamp': event['income-timestamp']}
```

Comment:
Leftover comments?

Reply:
I had a doubt to clarify. That's why I commented on it. When there is an exception, is it enough to return only {'result',' status' }? I noticed in some cases, that some values do exist and can be returned. For example, here, 'timestamp' can be returned in 'measurements'